bundles/core/src/main/java/com/adobe/cq/wcm/core/components/sandbox/models/Teaser.java

```diff
@@ -45,6 +45,27 @@
      */
     String NN_CTAS = "ctas";
```

Comment:
I find "ctas" quite unreadable if you're not familiar with the concept. Can't we simply refer to these as "actions".

`NN_ACTIONS="actions"`
`PN_ACTION_LINK="link"`
`PN_ACTION_TEXT="text"`
`PN_ACTION_DISABLED="disabled"`

Also, `disableCTA` sounds more like a directive than a state, `disabled` seems more suitable.

Reply:
The description also seems wrong here, shouldn't it be "Name of the child node where the Call-to-Actions are stored" (child rather than parent)

Reply:
`actions` is indeed easier to understand, I think it makes sense to replace `CTA(s)` with `action(s)`.

@jckautzmann also suggested we switch from directives to states for properties, we can rename them all to be consistent. We should also describe in the javadoc where the property is located (resource, style/content-policy).

The `actions` node is a parent of each action, yet a child of the teaser ;)

Reply:
+1 - we should review these all at the same time and ensure we're taking the same approach cross-component.